util/dvsim/utils.py

```diff
@@ -183,17 +183,17 @@ def find_and_substitute_wildcards(sub_dict,
     return sub_dict
 
 
-def md_results_to_html(title, css_path, md_text):
+def md_results_to_html(title, use_css_style_for_web, md_text):
     '''Convert results in md format to html. Add a little bit of styling.
     '''
     html_text = "<!DOCTYPE html>\n"
     html_text += "<html lang=\"en\">\n"
     html_text += "<head>\n"
```

Comment:
Should the styling be moved above `<head>`?

Reply:
The [example](https://www.w3schools.com/html/html_css.asp) here I find seems to put `style` after `head`. 
When I move the `<style>` in front of `<head>`, in email html format, the color code seems to be gone. Not sure if i missed any details.

Reply:
Ok nvm.